src/Compilers/Test/Utilities/CSharp/NullableAttributesVisitor.cs

```diff
@@ -46,19 +50,40 @@ public override void VisitNamespace(NamespaceSymbol @namespace)
 
         public override void VisitNamedType(NamedTypeSymbol type)
         {
+            var previousContext = _nullableContext;
+            _nullableContext = GetNullableContextAttribute(type.GetAttributes()) ?? _nullableContext;
+
             ReportSymbol(type);
             VisitList(type.TypeParameters);
-            VisitList(type.GetMembers());
+
+            foreach (var member in type.GetMembers())
+            {
+                // Skip accessors since those are covered by associated symbol.
```

Comment:
>// Skip accessors since those are covered by associated symbol. [](start = 16, length = 63)

It doesn't look like there is a special handling for that for events.